saws/saws.py

```diff
@@ -460,5 +460,5 @@ def run_cli(self):
         print('Version:', __version__)
         print('Theme:', self.theme)
         while True:
-            document = self.aws_cli.run()
+            document = self.aws_cli.run(reset_current_buffer=True)
```

Comment:
@jonathanslenders here's some additional context, I'm looking to update [gitsome](https://github.com/donnemartin/gitsome) to 1.0.0 prompt-toolkit.  gitsome is using a forked/bundled xonsh which I'd like to move to a dependency in the future.  Here's the get_input() method which seems to be the equivalent to run():

https://github.com/donnemartin/gitsome/blob/master/xonsh/prompt_toolkit_shell.py#L71-L79

I'm curious if there's something I should do there to fully support 1.0.0?

Thanks in advance!